src/inference/src/dev/make_tensor.cpp

```diff
@@ -338,6 +338,95 @@ class RoiTensor : public ITensor {
     const size_t m_offset;
 };
 
+/**
+ * @brief ROI tensor on other tensor
+ * ROI tensor holds the owner
+ */
+
+class RoiRemoteTensor : public IRemoteTensor {
```

Comment:
should it be also inherited from RoiTensor?
I suppose in this case some method don't need to be implemented explicitly

Reply:
I've added a new BaseRoiTensor class and inherited both RoiTensor and RoiRemoteTensor from it